packages/components/src/utils/unit-values.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Parses a number and unit from a value.
+ *
+ * @param  toParse Value to parse
+ *
+ * @return  The extracted number and unit.
+ */
+export function parseUnitValue(
+	toParse: string | number
+): [ number | string | undefined, string | undefined ] {
+	// eslint-disable-next-line eqeqeq
+	if ( toParse == null ) {
```

Comment:
I'd personally like to avoid disabling eslint rules inline when possible, as they tend to accumulate technical debt.

Could we disable it at a package level?

As an alternative, could we just switch to the slightly more verbose

```suggestion
	if ( toParse !== null && typeof toParse !== 'undefined' ) {
```

?